app/cli/src/main/scala/org/bitcoins/cli/ConsoleCli.scala

```diff
@@ -505,6 +561,12 @@ object CliCommand {
       amount: Bitcoins,
       satoshisPerVirtualByte: Option[SatoshisPerVirtualByte])
       extends CliCommand
+  case class SendFromOutPoints(
+      outPoints: Vector[TransactionOutPoint],
+      destination: BitcoinAddress,
+      amount: Bitcoins,
+      satoshisPerVirtualByte: Option[SatoshisPerVirtualByte])
```

Comment:
Do we want this to be available on segwit only? Isn't `sats/vbyte` only available on segwit outpoints?

Reply:
No, on non-segwit txs it will just be the same as sats/byte

Reply:
Ok, what about mixed outpoints? I.e. non segwit and segwit utxos? We probably need to come up with a coherent model for this case -- it's hard to think about!

Reply:
This is just handled by `FeeUnit.calc` should be handled correctly already

Reply:
Ok, so when doing our API, does fee unit of `sat/vbyte` or `sat/byte` matter? Can you pay a fee too large or too small of a fee depending on that unit, and how we interpret it?

Obviously if we were to interpret the number as `SatsPerKiloByte` will effect your fee rate, but the difference between `sats/vbyte` & `sats/byte` wont'?

Reply:
It's handled correctly, for `sats/vbyte` it does `tx.vsize * fee rate`, for `sats/byte` it does `tx.baseSize * fee rate`